core/utils.py

```diff
@@ -264,3 +264,15 @@ async def update_presence(bot: commands.Bot, *, status: str = None) -> None:
                 name=config.prefix + "help",
             ),
         )
+
+    @staticmethod
+    async def send_dm(
+        user: Union[discord.Member, discord.User],
+        text: Optional[str] = None,
+        embed: Optional[discord.Embed] = None,
```

Comment:
To follow the discord.py conventions, `embed` should be named argument (`send_dm(user, text, *, embed):`).